bridgesample/src/main/java/com/livefront/bridgesample/main/model/Scenario.kt

```diff
@@ -19,12 +19,23 @@ fun getScenarios(context: Context): List<MainItem> = listOf(
         MainItem(
                 R.string.large_data_scenario_title,
                 R.string.large_data_scenario_description,
-                LargeDataActivity.getNavigationIntent(context)
+                LargeDataActivity.getNavigationIntent(context, false)
         ),
         MainItem(
                 R.string.non_bridge_large_data_scenario_title,
                 R.string.non_bridge_large_data_scenario_description,
-                NonBridgeLargeDataActivity.getNavigationIntent(context)
+                NonBridgeLargeDataActivity.getNavigationIntent(context, false)
+        ),
+        MainItem(
+                R.string.large_data_backstack_scenario_title,
+                R.string.large_data_backstack_scenario_description,
+                LargeDataActivity
+                        .getNavigationIntent(context, true)
```

Comment:
Let's keep `getNavigationIntent` on the same line as above:

```kotlin
LargeDataActivity.getNavigationIntent(context, true)
```

Reply:
Also, for clarify let's use named parameters on all these booleans:

```kotlin
LargeDataActivity.getNavigationIntent(context, infiniteBackStack  = true)
```

etc.

Reply:
Done.